pod/perlfunc.pod

```diff
@@ -3773,8 +3773,10 @@ You should not use this function for rounding: one because it truncates
 towards C<0>, and two because machine representations of floating-point
 numbers can sometimes produce counterintuitive results.  For example,
 C<int(-6.725/0.025)> produces -268 rather than the correct -269; that's
-because it's really more like -268.99999999999994315658 instead.  Usually,
-the L<C<sprintf>|/sprintf FORMAT, LIST>,
+because it's really more like -268.99999999999994315658 instead.
+Similarly, C<int "Info"> produces Inf, see
+L<perldata/Special floating point: infinity (Inf) and not-a-number (NaN)>
```

Comment:
Is this really "similar" to the above? Floating point accuracy seems unrelated to how Perl doesn't force the result of `int` to be an integer.

While the link addresses the string conversion to a float, nothing in it or this text address how for numbers outside the integer range, perl will still return a float. And while mathematically `1e50` is an integer even if internally stored as a float, Inf and NaN are definitely not integers.

Reply:
Well, it's similar because it "can sometimes produce counterintuitive results". But I agree the wording could be improved.

Reply:
Updated with a fixup. Let me know if I should squash the changes.